### DISCUSSION TITLE
audio get amplified, don't know why

### DISCUSSION THREAD

**Discussion by tellierflexus** [baresip/baresip#2484] [Q&A]
Hello everyone, 
Using baresip I'm trying to achieve a setup where I can retrieve an audio from a remote radio studio. For that I've configured baresip to auto-answer, then output to a specific soundcard (L24, 48khz). But, when I'm sending a signal using this system, the audio get abnormally amplified. You can see on my screen the audio sent, on the left, and the same audio receive on the right.
<img width="94" alt="Capture d’écran 2023-03-05 à 23 36 00" src="https://user-images.githubusercontent.com/7304635/222989985-ef55757f-ba1c-4a8a-9176-202739ec88e9.png">

Here's a part of my config : 
`
# Audio
#audio_path             /usr/share/baresip
audio_player            alsa,default
audio_source            alsa,default
audio_alert             alsa,default
ausrc_srate             48000
auplay_srate            48000
ausrc_channels          2
auplay_channels         2
#audio_txmode           poll            # poll, thread
audio_level             no
ausrc_format            s16             # s16, float, ..
auplay_format           s16             # s16, float, ..
auenc_format            s16             # s16, float, ..
audec_format            s16             # s16, float, ..
audio_buffer            20-160          # ms
audio_buffer_mode       fixed           # fixed, adaptive
audio_silence           -35.0           # in [dB]
audio_telev_pt          101             # payload type for telephone-event
`

and the audio pipeline :
`
audio tx pipeline:        alsa ---> aubuf ---> auconv ---> opus 
audio rx pipeline:        alsa <--- aubuf <--- auconv <--- opus`

Moreover when trying to configure ausrc_format and auplay_format to s24_3le I obtained the following error : auconv: format not supported (S24_3LE)

Any idea ?

Have a good day :)

**Comment by cspiel1** [comment]
Baresip does not amplify audio. With `alsa <--- aubuf <--- auconv <--- opus` the amplification most likely is caused by your ALSA configuration.

Baresip runs with s16le internally. If you need other sample formats for MIC and speaker they have to be converted by module `auconv`. Currently only `float` and `s16le` are supported.
